pkgs/top-level/all-packages.nix

```diff
@@ -39436,4 +39436,6 @@ with pkgs;
   gnss-share = callPackage ../servers/gnss-share { };
 
   ali = callPackage ../tools/networking/ali { };
+
+  udict = callPackage ../tools/networking/udict { };
```

Comment:
This is not a networking package, but a dictionary that browses the internet. It should be on `applications/misc`.